pyplanet/apps/contrib/brawl_match/__init__.py

```diff
@@ -191,6 +191,13 @@ def __init__(self, *args, **kwargs):
 		for index, (uid, _) in enumerate(self.match_maps, start=1):
 			map_name = (await Map.get_by_uid(uid)).name
 			await self.brawl_chat(f'[{index}/{len(self.match_maps)}] {map_name}')
+		await self.check_for_wu_messages()
+
+	async def check_for_wu_messages(self):
+		if self.maps_played == len(self.match_maps) - 1:
+			await self.brawl_chat('$oNo more warmup in future!')
+		elif self.maps_played == len(self.match_maps):
+			await self.brawl_chat('$oWarmup is disabled!')
```

Comment:
Can we call `self.remove_wu()` here instead since this check is identical to our check to see if we should remove WU? Not sure about the flow here if calling it here would be too late. But i think i would prefer if we were able to call it from here, since we are saying we are saying its disabled now.

Reply:
Did this in commit cc35a08